### PR TITLE
Removed usage of remaining print-as-a-statement

### PR DESCRIPTION
**Pull request by claudep**
print 'as a statement' does not exist any longer in Python 3.

### CODE REVIEW

File: translate/search/match.py [hunk 1]
```diff
@@ -90,8 +90,8 @@ def inittm(self, stores, reverse=False):
         for store in stores:
             self.extendtm(store.units, store=store, sort=False)
         self.candidates.units.sort(key=sourcelen, reverse=self.sort_reverse)
-        # print "TM initialised with %d candidates (%d to %d characters long)" % \
-        #        (len(self.candidates.units), len(self.candidates.units[0].source), len(self.candidates.units[-1].source))
+        # print("TM initialised with %d candidates (%d to %d characters long)" % \
+        #        (len(self.candidates.units), len(self.candidates.units[0].source), len(self.candidates.units[-1].source)))
```
Comment (unho) [inline translate/search/match.py]: Just wondering if we should remove all this commented code.

**Comment by unho** [comment]
lgtm

File: translate/search/match.py (hunk 1 above)
Comment (claudep) [inline translate/search/match.py]: Maybe replaced by a logging call? Anyway, that could be another patch.

Comment (unho) [inline translate/search/match.py]: Sure.

**Comment by jleclanche** [comment]
You can probably remove most of these comments...

**Comment by claudep** [comment]
OK, patch rebased with commented prints removed.

**Comment by unho** [comment]
Merged. Thanks!
